lib/vehicle.js

```diff
@@ -747,4 +828,16 @@ Vehicle.prototype.request = async function(
  *   for all possible errors.
  */
 
+/**
+ * @name Vehicle#security
+ * @function
+ * @memberof Vehicle
+ * @description Returns the security status of the vehicle.
```

Comment:
"Returns the lock status of the vehicle."

I still interchange the names between lock and security. The operation is lock status but the endpoint and route is security. Usually in descriptions we refer to the lock status.

https://github.com/smartcar/api-schema/blob/master/src/paths/security.yaml